incubator/riemann/templates/service.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "riemann.fullname" . }}
+  labels:
+    app: {{ template "riemann.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+      {{- if .Values.service.ports.tcp }}
```

Comment:
I find it kind of weird that you indent the if statement instead of the code that is wrapped inside of it. Please do it the other way around. Apply everywhere.